knowledge/pkg/vectorstore/types/types.go

```diff
@@ -1,8 +1,58 @@
 package types
 
+import (
+	"slices"
+)
+
 type Document struct {
 	ID              string         `json:"id"`
 	Content         string         `json:"content"`
 	Metadata        map[string]any `json:"metadata"`
 	SimilarityScore float32        `json:"similarity_score"`
 }
+
+const (
+	DocMetadataKeyDocIndex  = "docIndex"
+	DocMetadataKeyDocsTotal = "docsTotal"
+)
+
+func mustInt(value any) int {
+	switch v := value.(type) {
+	case int:
+		return v
+	case float64:
+		return int(v)
+	default:
+		panic("Unsupported type")
```

Comment:
Are we sure it's safe to panic here?

Reply:
No.. this is mostly laziness at a point where I'm pretty sure we'll never hit that case 😅